pkg/netpol/diff/diff_formatter.go

```diff
@@ -215,3 +215,130 @@ func (cs *diffFormatCSV) writeDiffOutput(connsDiff ConnectivityDiff) (string, er
 func (cs *diffFormatCSV) singleDiffLine(d *singleDiffFields) string {
 	return fmt.Sprintf("%s;%s;%s;%s;%s;%s", d.diffType, d.src, d.dst, d.dir1Conn, d.dir2Conn, d.workloadDiffInfo)
 }
+
+// /////////////////////////
+// diffFormatDOT: implements the diffFormatter interface for dot output format
+type diffFormatDOT struct {
+}
+
+const (
+	dotHeader  = "digraph {"
+	dotClosing = "}"
+)
```

Comment:
we have these const value also defined in connlist pkg, consider moving this to common pkg?